include/yaclib/util/result.hpp

```diff
@@ -130,7 +130,6 @@ class Result {
         throw ResultEmpty{};
       }
     }
-    Unreachable();
```

Comment:
NIT: Maybe we should add some comment here for forthcoming researchers?

Reply:
No need, we will rewrite this class anyway.